person.rb

```diff
@@ -0,0 +1,26 @@
+# Person class
+class Person
+  def initialize(age, parent_permission = 'true', name = 'Unknown')
```

Comment:
- [ ] I think `parent_permission` should be of boolean type: `parent_permission = true`. Without the quotes. This will lead to a linter error. I'm sure you can find a solution for this on google :)

Reply:
Yes, you're right @D4nielJ. I will correct that immediately